<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Acer" failures="1" tests="2" name="Command line test" time="1.331" errors="0" timestamp="2021-09-24T15:39:41 COT">
  <testcase classname="test.TestGeneral" name="verificarCargaFundaciones" time="1.131">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at persistencia.FundacionesDAO.consultarFundaciones(FundacionesDAO.java:23)
at test.TestGeneral.verificarCargaFundaciones(TestGeneral.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- verificarCargaFundaciones -->
  <testcase classname="test.TestGeneral" name="verificarConexion" time="0.102"/>
</testsuite> <!-- Command line test -->
